kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/CertManager.java

```diff
@@ -55,7 +70,12 @@ public void deploy() {
      * @throws IOException the io exception
      */
     public void delete() throws IOException {
+        if (!deleteCertManager) {
+            LOGGER.warn("Skipping cert manager deletion. It was previously installed");
+            return;
```

Comment:
If the system test is the responsible of installing cert-manager, we will remove it using the same yaml file used for the installation, so the deletion will be done as expected. But if the cert-manager is already installed by another party, we won't delete it as it can be needed for other test cases.

Reply:
Yes I understand that but should we be removing any certs the system tests have created? So that we leave the cert-manager instance in the state we found it

Reply:
Which cert is kroxy creating? I didn't pay attention to that so that we where removing the cert-manager always...

Reply:
Kroxylicious doesn't create certs as far as I know but we are dealing with cert-manager for a reason here. Is it just for strimzi?

Reply:
Strimzi doesn't need cert-manager for running (right now). So I'm not sure for what it's used.

Reply:
So maybe we don't need it and we didn't know it...

Reply:
I'm going to create an issue to investigate/remove cert manager if it is not needed for kroxy system tests